qutebrowser/keyinput/modeparsers.py

```diff
@@ -29,9 +29,11 @@
 from qutebrowser.config import config
 from qutebrowser.keyinput import keyparser
 from qutebrowser.utils import usertypes, log, objreg, utils
+from qutebrowser.config.parsers import keyconf
 
 
-STARTCHARS = ":/?"
+conf = keyconf.KeyConfigParser(None, None) 
+STARTCHARS = conf.get_bindings_for('normal').keys()
```

Comment:
I'm not sure if this is going to work - I think it's fine to keep them hardcoded here, as the statusbar can still show `:`, `/` and `?` even if the key is rebound.